### ISSUE TITLE
Template glob patterns only publish a single template

### ISSUE THREAD

**Issue by lachlancollins** [stackblitz-labs/pkg.pr.new#136]
Hi there, I've noticed that supplying the publish command with a glob of templates only seems to output the first template it finds: https://github.com/TanStack/query/pull/7676

The GitHub workflow only logs that it is publishing the one template: https://github.com/TanStack/query/actions/runs/9805036328/job/27073907211?pr=7676

The readme shows this example:
`npx pkg-pr-new publish ./packages/A --template ./examples/*`

This is the command I am trying:
`pnpx pkg-pr-new publish --compact ./packages/* --template ./examples/react/*`

**Comment by Aslemammad** [comment]
I see, I guess there's an issue with that. Would you like to send a PR?

**Comment by Aslemammad** [comment]
Btw, in the new release that you sent a pr for, we do not skip the private templates anymore? So maybe you need to re-run the workflow again or update pkg.pr.new

![image](https://github.com/stackblitz-labs/pkg.pr.new/assets/37929992/6736edcd-6e31-4fb3-8390-2d83cb4a4922)

**Comment by lachlancollins** [comment]
Hi @Aslemammad , I've debugged the problem further. It's definitely running the latest version because the algolia example wouldn't run with the old version.

What seems to be currently happening is that the CLI itself (citty? pnpm?) is resolving the glob patterns, before they reached the args field.

If you run `pnpm pkg-pr-new publish ./packages/* --template templates/*` and log the value of args, it returns this:

```
{"_":["./packages/backend","./packages/cli","./packages/utils","templates/example-2"],"template":"templates/example-1","comment":"update"}
```

So it seems to be converting `templates/*` into `templates/example-1` and `templates/example-2`, then only passing the first one to the `--template` field.

This can be partially overcome by passing it as a string (`--template="templates/*"`), but this doesn't resolve at all.

I have a PR up to fix this problem: #137

**Comment by lachlancollins** [comment]
This seems to explain the unintended CLI behaviour: https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784

**Comment by Aslemammad** [comment]
I see, did that solve the issue for you? Let me know if I can help

**Comment by Aslemammad** [comment]
just released 0.0.14! I appreciate the hard work, Lachlan!

**Comment by lachlancollins** [comment]
> just released 0.0.14! I appreciate the hard work, Lachlan!

Thanks for your quick replies, this is working brilliantly!
